modules/attachments/main.tf

```diff
@@ -0,0 +1,19 @@
+resource "ibm_scc_profile_attachment" "scc_profile_attachment" {
+  profile_id  = var.profile_id
+  instance_id = var.scc_instance_id
+  name        = var.attachment_name
+  description = var.attachment_description
+  schedule    = var.attachment_schedule
+  status      = var.attachment_status
+  scope {
```

Comment:
It says [scope](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/scc_profile_attachment#scope) is a list with max size 8, what if they want a different scope for `dev`, `prod` etc. Can this example handle that? Does it also work if the user doesn't want to specify an environment?